### ISSUE TITLE
JavaScript Error Could Not Find DOM Element for Container

### ISSUE THREAD

**Issue by marktiddy** [htmlburger/carbon-fields#1065]
## Version

  - Carbon Fields:  3.3.2
  - WordPress: 5.8.2
  - PHP: 7.4

## Expected Behavior
_Please enter the expected behavior here_

## Actual Behavior
My custom post types should have come custom fields created using Container::make('post_meta')

## Container definition

        Container::make('post_meta','Testimonials Info')
            ->add_fields(array(
                Field::make('text','author',__('Author'))
                    ->set_required(true),
                Field::make('textarea','quote',__('Quote'))
                    ->set_required(true)
            ))
            ->where('post_type','=','testimonials')
 

## Steps to Reproduce the Problem

  1. Try to create a new page within wordpress
  2. A blank page appears with the attached screenshot of errors in the console
  3. Commenting out the above code fixes the issue

Image of errors: [Link to Screenshot](https://drive.google.com/file/d/1GWKc1UoCw-F7UCG0cu_gDHfMZZ9-GHrH/view?usp=sharing )

## Comments
The custom post type in question that I've declared has no archive and isn't set to show in rest. The fields do initially appear for my custom post type but it only seems to be after a custom post type is saved that there is then an issue with creating new pages. I've managed to replicate this issue on two sites and in both cases Carbon Fields is set up and called within functions.php after the post types have been set up.

**Comment by dhalucario** [comment]
My best guess is that some type of custom page editor removes the elements carbon fields looks for on the backend. Which plugins do you use?

**Comment by marktiddy** [comment]
Thanks. The site isn't running any custom page editors at all, the only plugins are Contact Form 7, Flamingo and WP SMTP none of which should have any impact on the editor

**Comment by marktiddy** [comment]
For anyone having the same issue I found the thing causing it for me.

I had a custom post type which had some carbon meta fields within them.

I also had some Carbon Field Gutenberg blocks. One of these blocks had a multi-select field which I was dynamically populating the options for using a custom WP_Query to get a list of custom post type IDs and names. I'm not sure why but this was causing some kind of conflict.

To resolve it I switched out the multi-select for an association field and the problem is resolved.
